projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.navigation.spec.ts

```diff
@@ -1081,105 +901,106 @@ describe('IgxHierarchicalGrid Smaller Child Navigation #hGrid', () => {
-        // parentCell.onFocus(null);
+        GridFunctions.focusCell(fixture, parentCell);
+
+        await wait(DEBOUNCE_TIME);
         fixture.detectChanges();
 
-        GridFunctions.simulateCellKeydown(parentCell, 'ArrowDown');
+        UIInteractions.triggerEventHandlerKeyDown('arrowdown', baseHGridContent, false, false, false);
+
+        await wait(DEBOUNCE_TIME);
         fixture.detectChanges();
 
         // last cell in child should be focused
         const childGrid = hierarchicalGrid.hgridAPI.getChildGrids(false)[0];
         const childLastCell =  childGrid.getCellByColumn(0, 'Col1');
+
         expect(childLastCell.selected).toBe(true);
-        expect(childLastCell.focused).toBe(true);
-    });
+        expect(childLastCell.active).toBe(true);
+    }));
 
     it('should navigate to last cell in next row for child grid using Arrow Up from last cell of parent with more columns', (async () => {
         hierarchicalGrid.verticalScrollContainer.scrollTo(2);
         await wait(DEBOUNCE_TIME);
         fixture.detectChanges();
 
         const parentCell = hierarchicalGrid.getCellByColumn(2, 'Col2');
-        // parentCell.onFocus(null);
-        fixture.detectChanges();
+        GridFunctions.focusCell(fixture, parentCell);
 
-        GridFunctions.simulateCellKeydown(parentCell, 'ArrowUp');
         await wait(DEBOUNCE_TIME);
         fixture.detectChanges();
 
-        // last cell in child should be focused
-        const childGrids =  fixture.debugElement.queryAll(By.directive(IgxChildGridRowComponent));
-        const childGrid = childGrids[1].query(By.directive(IgxHierarchicalGridComponent)).componentInstance;
-        const childLastCell =  childGrid.getCellByColumn(9, 'ProductName');
-        expect(childLastCell.selected).toBe(true);
-        expect(childLastCell.focused).toBe(true);
-    }));
+        UIInteractions.triggerEventHandlerKeyDown('arrowup', baseHGridContent, false, false, false);
 
-    it('should navigate to last cell in previous row for child grid using Shift+Tab from parent with more columns', (async () => {
-        hierarchicalGrid.verticalScrollContainer.scrollTo(2);
         await wait(DEBOUNCE_TIME);
         fixture.detectChanges();
 
-        const parentCell = hierarchicalGrid.getCellByColumn(2, 'ID');
-        // parentCell.onFocus(null);
-        fixture.detectChanges();
-
-        GridFunctions.simulateCellKeydown(parentCell, 'Tab', false, true);
-        await wait(DEBOUNCE_TIME);
-        fixture.detectChanges();
-
-        const childGrids =  fixture.debugElement.queryAll(By.directive(IgxChildGridRowComponent));
-        const childGrid = childGrids[1].query(By.directive(IgxHierarchicalGridComponent)).componentInstance;
+        // last cell in child should be focused
+        const childGrids =  fixture.debugElement.queryAll(By.css('igx-child-grid-row'));
+        const childGrid = childGrids[1].query(By.css('igx-hierarchical-grid')).componentInstance;
```

Comment:
Can you revert the original code to get the Grids by directive or create constants for the:
igx-child-grid-row
igx-hierarchical-grid